tools/rapids-package-name

```diff
@@ -29,6 +29,11 @@ case "${pkg_type}" in
     append_pyver=1
     append_arch=1
     ;;
+  wheel_cpp)
+    append_wheelname=1
+    append_cuda=1
```

Comment:
It doesn't look like we append the CUDA version for `wheel_python`. Is that because we're using wheel names that include the suffix like `-cu12`? We should probably mirror that behavior for `wheel_cpp`.